Inventory/Test/Integration/Stock/StockSourceLinkProcessorTest.php

```diff
@@ -54,6 +73,56 @@ protected function setUp(): void
      * @magentoDataFixture Magento_InventoryApi::Test/_files/stock_source_links.php
      */
     public function testProcess()
+    {
+        $this->auth->login(
+            \Magento\TestFramework\Bootstrap::ADMIN_NAME,
+            \Magento\TestFramework\Bootstrap::ADMIN_PASSWORD
+        );
+
+        $linksData = $this->getLinksData();
+        $stockId = 10;
+
+        $this->stockSourceLinkProcessor->process($stockId, $linksData);
+
+        $sortOrder = $this->sortOrderBuilder
+            ->setField(StockSourceLinkInterface::PRIORITY)
+            ->setAscendingDirection()
+            ->create();
+        $searchCriteria = $this->searchCriteriaBuilder
+            ->addFilter(StockSourceLinkInterface::STOCK_ID, $stockId)
+            ->addSortOrder($sortOrder)
+            ->create();
+        $searchResult = $this->getStockSourceLinks->execute($searchCriteria);
+
+        self::assertCount(2, $searchResult->getItems());
+    }
+
+    /**
+     * @magentoDataFixture Magento_InventoryAdminUi::Test/Integration/_files/user_assigned_to_stocks.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/sources.php
```

Comment:
Please, use relative path instead of absolute: `Magento_InventoryApi::Test/_files`

Reply:
@sidolov fixed